src/main/java/seedu/smartnus/logic/commands/DeleteCommand.java

```diff
@@ -22,9 +22,9 @@
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the item identified by the index number used in the respected displayed list.\n"
-            + "Parameters: ITEM_NAME: list or note \n"
-            + "INDEX (must be a positive integer)\n"
+            + ": Deletes the item identified by the index number used in the respective displayed list.\n"
+            + "Parameters: list / note \n"
+            + "INDEX (must be a positive integer between 1 and 2147483647)\n"
```

Comment:
Perhaps you can include that it's inclusive here as well? Or maybe you can standardise the messages, like above you said it's "between 1 and 2147483647 (both inclusive)", and below it's " from 1 to 2147483647"

Reply:
Good point. Updated the message.